src/Android/Avalonia.Android/Platform/Input/AndroidInputMethod.cs

```diff
@@ -89,6 +89,8 @@ public void SetClient(TextInputMethodClient? client)
             }
             else
             {
```

Comment:
Shouldn't `_inputConnection` be set to null directly here rather than waiting for the init callback to called again?